### ISSUE TITLE
npm install -> module not found in registry...

### ISSUE THREAD

**Issue by jeffleus** [awslabs/s3-managed-download-js#8]
OK, editing to start w the basics.  I tried to run the npm install and the package is not found '@aws/s3-managed-download'.  Is this not published yet?

npm ERR! 404 Not found : @aws/s3-managed-download
npm ERR! 404 
npm ERR! 404  '@aws/s3-managed-download' is not in the npm registry.

**Comment by jeffleus** [comment]
OK, I went ahead and forked the repo and built/published my own version of the module to npm, jeffleus-managed-downloader.  Then, I included it in a new project and successfully ran a multipart download.  As part of that work, I noticed a bug in the project name vs the documentation for the npm install.  I will try and complete a pull request to sync this up.  I am also interested in what it would take to add an EventEmitter to the model similar to the ManagedUploader in s3-sdk.  It would be great to get 'progress' events to update a UI when pulling down content.  I will create a separate issue for this.
